src/patternfly/components/DataList/data-list.scss

```diff
@@ -34,7 +44,7 @@ $pf-data-list--breakpoint--mobile: 380px;
   --pf-c-data-list__cell-cell--md--PaddingTop: var(--pf-global--spacer--xl);
   --pf-c-data-list__cell-cell--md--MarginLeft: var(--pf-global--spacer--xl);
 
-  @media screen and (min-width: $pf-global--breakpoint--md) {
+  @media screen and (min-width: $pf-data-list--breakpoint--mobile) {
```

Comment:
Shouldn't this match the breakpoint on the page content and pagination components where the left/right padding changes?

Reply:
Yes it should, I had changed it to use `$pf-global--breakpoint--md` to match pagination/page breakpoints but I didn't want to make this PR a breaking change.. what are your thoughts?